src/core/public/chrome/chrome_service.tsx

```diff
@@ -115,6 +116,7 @@ export interface StartDeps {
   application: InternalApplicationStart;
   docLinks: DocLinksStart;
   http: HttpStart;
+  // core: CoreStart;
```

Comment:
Better not contain annotation in component file.

Reply:
Thank you, updated.